packages/protocol/test/governance/validators.ts

```diff
@@ -2360,12 +2337,6 @@ contract('Validators', (accounts: string[]) => {
         assert(parsedGroup.lastSlashed > initialTimestamp)
       })
     })
-
-    describe('when called from an unapproved address', async () => {
```

Comment:
Why remove this?

Reply:
Change from https://github.com/celo-org/celo-monorepo/pull/2347